### ISSUE TITLE
Branding issue - Don't Die / No Don't Die

### ISSUE THREAD

**Issue by davidwolinsky** [giles-v/ghost-theme-dont-die#7]
Less a feature request but a design headache that really isn't a big deal. People think the name of the project is the URL instead off what's all over the page elsewhere. Just something I wonder if we can drill into people's heads and something I always forget to mention. It comes up sometimes, and if you have suggestions as we push into 2.0 would love to scoop this up with what we deploy down the road.

**Comment by giles-v** [comment]
I think the domain is the problem. Which is a pity, because I can't think of a better one. I don't think there's a good way to handle two confusing brand handles.

It's not just a perception problem. If I need to share the details with someone, I'm not going to say, "the name of the project is 'don't die', but you have to put 'nodontdie' as the URL". I'm simply going to say, 'check out nodontdie.com'. And that will be their first impression of the branding. That's not a communication failure, that's simply the URL being the most important sharable brand point, and it's catchy.

**Comment by davidwolinsky** [comment]
Yeah. It really isn't a big deal, but figured as long as I'm uploading my
brain to you -- just something I wondered about. Unfortunately, from the
outset, someone has sat on dontdie.com. Doesn't feel like a great use of
time, money, energy to get in a bidding war. Not mission critical, but
something I wanted to mention!
